src/test/datascience/uiTests/helpers.ts

```diff
@@ -95,7 +96,9 @@ export class BaseWebUI implements IAsyncDisposable {
             ? setTimeout(async () => {
                   if (!promise.resolved) {
                       // Take a screenshot and save at the root with the same name (upload on error)
-                      await this.page?.screenshot({ path: path.join(EXTENSION_ROOT_DIR, 'browser-screenshot.png') });
+                      await this.page
+                          ?.screenshot({ path: path.join(EXTENSION_ROOT_DIR, 'browser-screenshot.png') })
```

Comment:
Was throwing a lot of unhandled exceptions (page was already disposed at this point)